docs/src/rules/no-underscore-dangle.md

```diff
@@ -10,9 +10,11 @@ As far as naming conventions for identifiers go, dangling underscores may be the
 var _foo;
 ```
 
-There is actually a long history of using dangling underscores to indicate "private" members of objects in JavaScript (though JavaScript doesn't have truly private members, this convention served as a warning). This began with SpiderMonkey adding nonstandard methods such as `__defineGetter__()`. The intent with the underscores was to make it obvious that this method was special in some way. Since that time, using a single underscore prefix has become popular as a way to indicate "private" members of objects.
```

Comment:
I'd like to keep this history -- there's no harm in providing it and I think it provides helpful context both for the convention and for the rule. We can probably just change the last sentence to read like this:

> Since that time, using a single underscore prefix became a popular way to indicate that an object member was to be considered "private" and not part of its public interface.